index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="styles.css">
+  <title>Luis's Portfolio</title>
+</head>
+<body>
+  <div class="wrapper">
+    <header>
+      <nav class="flex-nav">
+        <ul>
+          <li class="logo"> <a href="#">My Logo</a></li>
+          <li class="menu-item">Portfolio</li>
+          <li class="menu-item">About</li>
+          <li class="menu-item">Contact</li>
+          <li class="hamburger-menu">☰</li>
```

Comment:
- [ ] As the previous code reviewer 🤓 , I will also recommend wrapping your hamburger button inside a `<nav>` tag to follow HTML/CSS best practices.

Reply:
Hello! Glad this came up. I had done the correction per the last code reviewer. However, this led to a series of errors in the linter stating a `nav` tag can't be inside an `ul`. How should I address this? I also thought of removing the hamburger menu from this element to be in its own `nav` tag but considering the `ul` is my flex item, this would involve changing the entire structure I have in place.